apps/vikunja/docker-compose.yml

```diff
@@ -18,46 +18,25 @@ services:
       - tipi_main_network
     labels:
       runtipi.managed: true
-  vikunja-api:
-    container_name: vikunja-api
-    image: vikunja/api:0.22.1
+  vikunja:
+    container_name: vikunja
+    image: vikunja/vikunja:0.24.1
     environment:
       VIKUNJA_DATABASE_HOST: vikunja-db
       VIKUNJA_DATABASE_PASSWORD: ${VIKUNJA_DB_PASSWORD}
       VIKUNJA_DATABASE_TYPE: postgres
       VIKUNJA_DATABASE_USER: tipi
       VIKUNJA_DATABASE_DATABASE: vikunja
       VIKUNJA_SERVICE_JWTSECRET: ${VIKUNJA_SERVICE_SECRET}
-      VIKUNJA_SERVICE_FRONTENDURL: ${APP_PROTOCOL:-http}://${APP_DOMAIN}/
+      VIKUNJA_SERVICE_PUBLICURL: ${APP_PROTOCOL:-http}://${APP_DOMAIN}/
     volumes:
       - ${APP_DATA_DIR}/data/files:/app/vikunja/files
     restart: unless-stopped
+    ports:
+      - ${APP_PORT}:3456
     depends_on:
       vikunja-db:
         condition: service_healthy
-    networks:
-      - tipi_main_network
-    labels:
-      runtipi.managed: true
-  vikunja:
-    image: vikunja/frontend:0.22.1
-    restart: unless-stopped
-    container_name: vikunja
-    networks:
-      - tipi_main_network
-    labels:
-      runtipi.managed: true
-  vikunja-proxy:
-    image: nginx
-    container_name: vikunja-proxy
-    ports:
-      - ${APP_PORT}:80
-    volumes:
-      - ${APP_DATA_DIR}/data/proxy/nginx.conf:/etc/nginx/conf.d/default.conf:ro
-    depends_on:
-      - vikunja-api
-      - vikunja
-    restart: unless-stopped
     networks:
       - tipi_main_network
     labels:
```

Comment:
Can you also update ` traefik.http.services.vikunja.loadbalancer.server.port: 80` to use port 3456?

Reply:
Done